1209/ABCD/Index.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: CodeBridge
+owner: Intelectron India
```

Comment:
Owner information is updated to intelectron.in (from Intelectron India) as suggested